lib/server.coffee

```diff
@@ -76,20 +79,25 @@ class Server
     @_setupGetRoute path, deployedHandler
 
 
-  _setupPutAndPostRoute: (path, handler) ->
+  _setupPutAndPostRoute: (path, handler, timeout) ->
     @server.route
       method: ['PUT', 'POST']
       path: path
       config:
         payload:
           maxBytes: MAX_PAYLOAD_BYTES
+        timeout:
+          server: timeout or null
```

Comment:
can we get a comment justifying the null case. Probably just an overall comment for the entire method.